compat/test/browser/suspense.test.js

```diff
@@ -1706,25 +1761,141 @@ describe('suspense', () => {
 
 		rerender();
 
-		/**
-		 * These currently failing assertion shows the issue that we currently unmount
-		 * the suspended tree (unlike react, which adds a display="none") and block any
-		 * further processing on that tree. Thus updates below a suspended Suspense are
-		 * getting lost.
-		 */
 		expect(log).to.eql(['construct', 'render', 'render']);
-
-		/**
-		 * When the above assertion will hold true we will certainly run into the second issue
-		 * here. The problem is that we do not remove suspensions from an instance of Suspense
-		 * when one of its suspending children no longer throws because of a state
-		 * update.
-		 */
 		expect(scratch.innerHTML).to.eql(
 			'<section><div>Suspender un-suspended</div></section>'
 		);
 	});
 
+	it('should allow multiple suspended children to update', () => {
+		function createSuspender() {
+			let suspender;
+			class Suspender extends Component {
+				constructor(props) {
+					super(props);
+					this.state = { promise: new Promise(() => {}) };
+					suspender = this;
+				}
+
+				unsuspend(content) {
+					this.setState({ promise: null, content });
+				}
+
+				render() {
+					if (this.state.promise) {
+						throw this.state.promise;
+					}
+
+					return this.state.content;
+				}
```

Comment:
Haven't seen this pattern before. Is this something you do in your own apps or just a quick way to write a test? Specifically referring to throwing a promise that never resolves. Would just like to better understand this use case 😅

Reply:
Well, it was copied from the test case "should allow suspended children to update", which was written by you 🤣

I believe it's just a quick wat to write the test.

Reply:
OH! Ahahahaha 😂😂😊 Got it, thanks :)

I'm thinking we may need to re-evaluate if this use case is really necessary at this moment (unsuspending without resolving the Promise). [React doesn't support it](https://codesandbox.io/s/resume-with-rerender-kim18) and currently it requires that we override the render method of a developer's component which feels icky. I know we do this for `componentWillUnmount` already, but its not a great pattern and one we should move away from (an `options.unmount` plugin would be better). We'll likely revisit this later. Feel free to leave your thoughts on PR #2917 😄

Reply:
Hmmm.. I wonder will adding options.commit and options.unmount be in the plan for Preact X, i'd love to help adding them in